R/plots-gi.R

```diff
@@ -1 +1,121 @@
 # Genetic interaction calculation step related plots can go here
+
+library(tidyverse)
+
+## This plot is meant to be functionally equivalent to Fig S5K (for HeLa, equivalent of Fig 3a for PC9).
+#From the figure's caption:
+#Scatter plot of target-level observed versus expected CRISPR scores in the HeLa screen.
+#The solid line is the linear regression line for the negative control (single KO) pgRNAs, 
+#while dashed lines indicate ± 2 residuals.
+
+plot_exp_v_obs_scatter <- function(gimap_dataset, facet_rep = TRUE, reps_to_drop = c("Day05_RepA_early")){
```

Comment:
```suggestion
plot_exp_v_obs_scatter <- function(gimap_dataset, facet_rep = TRUE, reps_to_drop = c("Day05_RepA_early")){
```

I understand why you have this right now, but well need to get rid of this default for `reps_to_drop` since it only pertains to one dataset.

Reply:
What suggestion do you have instead? Just add in the documentation or vignette an example of dropping reps before sending the data to the function?

Reply:
Oh, sorry, I misread the first message. I read it as I should get rid of the argument since some experiments will be treatments rather than timepoints. Not that the default there was specific to one timepoint experiment. Would you like the argument itself to stay?

Reply:
Ah no. Keep the argument, just get rid of the default. You'll have to see which was is easier. Probably "" should be able to work? I'm not sure.